pkg/storage/utils/eosfs/upload.go

```diff
@@ -26,6 +26,7 @@ import (
 	"strconv"
 
 	provider "github.com/cs3org/go-cs3apis/cs3/storage/provider/v1beta1"
+	"github.com/cs3org/reva/internal/http/services/owncloud/ocdav"
```

Comment:
Conceptually, do you thing ocdav is the correct place to have the content length header? It seems to be that it should be in some less specific place, used both inside the ocdav code and here.